src/scripts/default.conf

```diff
@@ -0,0 +1,5 @@
+{"OPERAND_COUNT_MIN": 2,
+ "OPERAND_COUNT_MAX": 11,
+ "TEST_COUNT": 100,
+ "SEED": 10000
+}
```

Comment:
Isn't this `default.conf` file contradicts with #7? @loki04 which implementation should we prefer?

Reply:
Use the `src/scripts/constants.py` for default setting.
And you can extend it if there is a need.